features/step_definitions/helper_services.rb

```diff
@@ -153,19 +153,48 @@
   step %Q/evaluation of `pod` is stored in the :proxy_pod clipboard/
 end
 
-Given /^I have LDAP service in my project$/ do
-    ###
-    # The original idea is trying to put ldap server in the openshift to make this flexy.
-    # Since we run the scenario in jenkins slave which is not in sdn, then two choices come to me:
-    # 1, Create a route for the ldapserver pod, but blocked by this us https://trello.com/c/9TXvMeS2 is done.
-    # 2, Port forward the ldap server pod to the jenkins slave.
-    # So take the second one since this one can be implemented currently
-    ###
+###
+# The original idea is trying to put ldap server in the openshift to make this flexy.
+# Since we run the scenario in jenkins slave which is not in sdn, then two choices come to me:
+# 1, Create a route for the ldapserver pod, but blocked by this us https://trello.com/c/9TXvMeS2 is done.
+# 2, Port forward the ldap server pod to the jenkins slave.
+# So take the second one since this one can be implemented currently
+###
+Given /^I have LDAP service in my project with clusterversion "([^"]*)"$/ do |clusterversion|
+  if clusterversion == '4.5' || '4.6'
```

Comment:
@rhpmali what if user put OCP < 4.6?  So I'm not sure if this scales.

Reply:
@pruan-rht If user put OCP < 4.6? it will execute the else loop and will work accordingly. So I have tested this with OCP version 4.3, 4.5 and 4.6 and it worked successfully

Reply:
@rhpmali sorry, that was a typo, I mean OCP > 4.6...

Reply:
@rhpmali how about `clusterversion.to_f <= 4.5`